src/components/date-picker/calendar.tsx

```diff
@@ -98,6 +99,31 @@ export class Calendar extends React.Component<CalendarProps, CalendarState> {
         }
     }
 
+    private getCalendarView = () => {
+        if (this.state.showYearView) {
+            return (
+                <div className="year-view" data-automation-id="YEAR_VIEW">
```

Comment:
There's a mix of camelCase and kebab-case class names in the stylesheet, I think our convention is to use camelCase.